scripts/metavar.py

```diff
@@ -289,7 +292,11 @@ def valid_value(self, test_value, error=False):
                 pass
         elif self.type is bool:
             if isinstance(test_value, str):
-                valid_val = (test_value in ['True', 'False']) or (test_value.lower() in ['t', 'f', '.true.', '.false.'])
+                if test_value.lower() in VariableProperty.__true_vals + VariableProperty.__false_vals:
+                    valid_val = (test_value.lower() in VariableProperty.__true_vals) or \
+                                not (test_value.lower() in VariableProperty.__false_vals)
```

Comment:
Does the second clause of the `or` operator ever change the value given its position inside the `if` statement above? Why not just omit it?

Reply:
Agreed, will change it